packages/components/src/spacer/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as Spacer } from './component';
+export * from './hook';
```

Comment:
The export could be more specific like in other places:

```suggestion
export { useSpacer } from './hook';
```